docs/releases.md

```diff
@@ -1,39 +1,157 @@
-# go-ipfs releases
+# `go-ipfs` Release Flow
 
-## Release Schedule
-go-ipfs is on a six week release schedule. Following a release, there will be
-five weeks for code of any type (features, bugfixes, etc) to be added. After
-the five weeks is up, a release candidate is tagged and only important bugfixes
-will be allowed up to release day.
+## Table of Contents
 
-## Release Candidate Checklist
-- [ ] CHANGELOG.md has been updated
-  - use `./bin/mkreleaselog` to generate a nice starter list
+- [Release Philosophy](#release-philosophy)
+- [Release Flow](#release-flow)
+- [Performing a Release](#performing-a-release)
+- [Release Version Numbers (aka semver)](#release-version-numbers-aka-semver)
+
+## Release Philosophy
+
+`go-ipfs` aims to have release every six weeks, two releases per quarter. During these 6 week releases, we go through 4 different stages of Release Candidates (RC) that gives us the opportunity to test the new version against our test envinronments, production envinronments, IPFS apps (e.g. Desktop and WebUI) and with our close partners that have IPFS running in Production, by leveraging their own test infrastructure.
+
+We might expand the six week release schedule in case of:
+- No new updates to be added
+- In case of a large community event that takes the core team availability away (e.g. IPFS Conf, Dev Meetings, IPFS Camp, etc)
+
+## Release Flow
+
+`go-ipfs` releases come in 4 stages:
+
+- **Release Stage 1 - Internal testing** - Test the Release against our testing infrastructure, including interoperability, integration, test lab, multiple runtimes and the apps we've built (WebUI, Desktop, NPM on IPFS, HTTP Client Libraries). The intent is to make this stage fully automated (and somewhat is already), until then, we manually check a list and ensure that all tests have been run
+- **Release Stage 2 - Invite IPFS partners to test** - Reach out to our partners (i.e. projects that have volunteered to support `go-ipfs` by using their own test infrastructure and tell us the results)
```

Comment:
Drawing from the above comments, I'd suggest changing the word _partners_ to _early testers_.